Source/Layout/BrickFlowLayout.swift

```diff
@@ -284,7 +298,27 @@ extension BrickFlowLayout {
         return contentSize
     }
 
+    func updateDirtyBricks(updatedAttributes: @escaping OnAttributesUpdatedHandler) {
+        for (section, item) in dirtyMap {
+            // This can be unwrapped safely
+            // because the sections that are in the dirtyMap should be there
```

Comment:
`should` what happens if they aren't there

Reply:
It's not possible that they aren't because they are set within below function. So there is never a case that they aren't there

`open override func shouldInvalidateLayout(forPreferredLayoutAttributes preferredAttributes: UICollectionViewLayoutAttributes, withOriginalAttributes originalAttributes: UICollectionViewLayoutAttributes) -> Bool`